src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -97,6 +99,26 @@ namespace winrt::TerminalApp::implementation
         _filteredActionsView().SelectionChanged({ this, &CommandPalette::_selectedCommandChanged });
     }
 
+    // Method Description:
+    // - Attempts to load user locale, to further use it when sorting the actions in the filtered view
+    // Return Value:
+    // - The user level locale, upon failure returns a default locale
+    std::locale CommandPalette::_loadUserLocale()
+    {
+        // Loading user locale name explicitly using GetUserDefaultLocaleName,
+        // as std:locale("") crashes on some setups
+        wchar_t userLocaleName[LOCALE_NAME_MAX_LENGTH] = { 0 };
+        auto localeNameSize = GetUserDefaultLocaleName(userLocaleName, ARRAYSIZE(userLocaleName));
+        if (localeNameSize == 0)
+        {
+            LOG_LAST_ERROR();
+            return std::locale();
+        }
+
+        std::wstring loadedLocale(userLocaleName);
```

Comment:
nit: `std::wstring loadedLocale{ userLocaleName };`

(we prefer using curly braces for variable initialization)